test/no-ignored-test-files.js

```diff
@@ -79,20 +83,20 @@ test('without AVA config in package.json', () => {
 			},
 			{
 				code: code(true),
-				filename: toPath('lib/foo.test.js'),
-				errors: [{message: 'Test file is ignored because it is not in `test.js test-*.js test/**/*.js`'}]
+				filename: toPath('lib/foo.spec.js'),
+				errors: [{message: 'Test file is ignored because it is not in `test.js test-*.js test/**/*.js **/__tests__/** **/*.test.js`'}]
```

Comment:
You didn't see anything....

![penguin-oops-XItRQJP0wai7m](https://camo.githubusercontent.com/58d9af6664c149ba3becc83b29848fb458c4c1c7/68747470733a2f2f6d65646961332e67697068792e636f6d2f6d656469612f58497452514a5030776169376d2f67697068792e676966)